viewer/viewer-awt/src/main/java/org/icepdf/ri/common/views/annotations/MarkupAnnotationPopupMenu.java

```diff
@@ -245,6 +274,18 @@ public void actionPerformed(ActionEvent e) {
 //            controller.setDocumentToolMode(DocumentViewModel.DISPLAY_TOOL_SELECTION);
             new FreeTextAnnotationHandler(controller.getDocumentViewController(), pageViewComponent)
                     .createFreeTextAnnotation(point.x, point.y - fontSize, false);
+        } else if (source == setAllPrivateMenuItem) {
+            ((SwingController) controller).changeAnnotationsPrivacy(a -> a instanceof MarkupAnnotation &&

```

Comment:
I'm not a huge fan of this pattern, I find it a bit hard to read but I'm sure I'll get better at it.  Would it make sense to create an inner class to capture the duplicate title==user_Name code?  Also are you planning to use invertFilter() in the interface at some point.

Reply:
Yeah I more or less was taught to use these in Java since the beginning and also did some functional programming, so I've no trouble reading these, but I can understand if you're not used to them.      
It does make sense to create an inner class in this case indeed, so I just did that.      
invertFilter is used in another pull request (for filtering annotations on the page through the MarkupAnnotationPanel)